lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/initial-journey-selection.yaml

```diff
@@ -34,6 +34,8 @@ CHECK_EXISTING_IDENTITY:
     fail-with-ci:
       targetJourney: FAILED
       targetState: FAILED
+    fail-with-ci-and-forced-reset:
+      targetState: RESET_VCS
```

Comment:
This needs to be `RESET_FAILED_IDENTITY` now

We could consider adding a test to `JourneyMapTest` to catch this sort of thing, but idk if it's worth it.